Fixing #24 (#31)
secure-systems-lab/peps: pep-0458.txt

=======================================

From time to time either a project or a version of a package will need to be revoked.

> To revoke trust in either a project or a version of a package, the bin role can 

Author: ```suggestion
To revoke trust in either a project or a version of a package, the associated bin-n role can 
```

> simply remove the delegation and re-sign the bin metadata.  This action only 

Author: ```suggestion
simply remove the delegation and re-sign the bin-n metadata.  This action only 
```

> simply remove the delegation and re-sign the bin-n metadata.  This action only 

Reviewer: What does *bin-n* delegate to? Shouldn't it be something like...
```suggestion
simply remove the corresponding targets and re-sign the bin-n metadata.  This action only 
```

Author: Agreed, good catch!

> requires actions with the online bin key.

Author: ```suggestion
requires actions with the online bin-n key.
```
